EXILED/Exiled.API/Features/Room.cs

```diff
@@ -507,6 +507,9 @@ private void InternalCreate()
                 Log.Error($"[ROOMTYPE UNKNOWN] {this} Name : {gameObject?.name} Shape : {Identifier?.Shape}");
 #endif
 
+            if (RoomLightControllersValue.Count == 0)
+                RoomLightControllersValue = new List<RoomLightController>();
+
```

Comment:
Can you explain me this change because i don't see what it's does make different ?